wordpress-mobile/gutenberg-mobile
# Issue/426 fix enter on captions

Fixes #426 

<img src="https://user-images.githubusercontent.com/651601/83859978-99392680-a716-11ea-809f-56afd74e1c15.gif" width=300>

To test:
 - Start the demo app
 - Add an image block
 - Set an image to it
 - Add a caption
 - When writing on the caption tap Enter on the end of the caption text
 - Check if new paragraph block is inserted
 - Do the same test as above using a video block.

PR submission checklist:
